From 26d83c69cc7cdfd1f05d9ae75d291810ca357781 Mon Sep 17 00:00:00 2001
From: Loren Huang <b02279@freescale.com>
Date: Fri, 14 Sep 2012 17:19:46 +0800
Subject: [PATCH 350/505] ENGR00224338 mx6:gpu:Remove modprobe check in gpu
 unit test

-As GPU kernel driver is changed to build-in driver, remove
modprobe check in gpu unit test.
-Correct document for unit test.

Signed-off-by: Loren Huang <b02279@freescale.com>
---
 test/gpu/doc/FSL-UT-GPU-001.txt | 2 +-
 test/gpu/gpu.sh                 | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/test/gpu/doc/FSL-UT-GPU-001.txt b/test/gpu/doc/FSL-UT-GPU-001.txt
index 50c1182..30db88d 100644
--- a/test/gpu/doc/FSL-UT-GPU-001.txt
+++ b/test/gpu/doc/FSL-UT-GPU-001.txt
@@ -17,7 +17,7 @@ GPU function test
 NO
 
 | Kernel Config Option |
-CONFIG_MXC_GPU=m
+CONFIG_MXC_GPU_VIV=y
 
 | Software Dependency |
 N/A
diff --git a/test/gpu/gpu.sh b/test/gpu/gpu.sh
index a5e2b95..3b49348 100755
--- a/test/gpu/gpu.sh
+++ b/test/gpu/gpu.sh
@@ -8,8 +8,8 @@ pushd .
 #
 #run modprobe test
 #
-gpu_mod_name=galcore.ko
-modprobe_test $gpu_mod_name
+#gpu_mod_name=galcore.ko
+#modprobe_test $gpu_mod_name
 #
 #run tests
 #
@@ -21,7 +21,7 @@ cd /opt/viv_samples/hal/ && ./tvui
 #
 #remove gpu modules
 #
-rmmod $gpu_mod_name
+#rmmod $gpu_mod_name
 #restore the directory
 #
 popd
-- 
1.8.0

